src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1530,17 +1531,21 @@ void window_guest_stats_paint(rct_window* w, rct_drawpixelinfo* dpi)
     y += LIST_ROW_HEIGHT;
 
     // Intensity
+    auto ft = Formatter::Common();
     auto maxIntensity = peep->intensity.GetMaximum();
-    set_format_arg(0, uint16_t, maxIntensity);
     int32_t string_id = STR_GUEST_STAT_PREFERRED_INTESITY_BELOW;
     if (peep->intensity.GetMinimum() != 0)
     {
-        set_format_arg(0, uint16_t, peep->intensity.GetMinimum());
-        set_format_arg(2, uint16_t, maxIntensity);
+        ft.Add<uint16_t>(peep->intensity.GetMinimum());
+        ft.Add<uint16_t>(maxIntensity);
         string_id = STR_GUEST_STAT_PREFERRED_INTESITY_BETWEEN;
         if (maxIntensity == 15)
             string_id = STR_GUEST_STAT_PREFERRED_INTESITY_ABOVE;
     }
+    else
+    {
+        ft.Add<uint16_t>(maxIntensity);
+    }
```

Comment:
Alternatively
```
    if (peep->intensity.GetMinimum() != 0)
    {
        ft.Add<uint16_t>(peep->intensity.GetMinimum());
        string_id = STR_GUEST_STAT_PREFERRED_INTESITY_BETWEEN;
        if (maxIntensity == 15)
            string_id = STR_GUEST_STAT_PREFERRED_INTESITY_ABOVE;
    }
    ft.Add<uint16_t>(maxIntensity);
```

(I'd also use a ternary to simplify the `maxIntensity` stuff, but out of scope